uwmweb/uwmcms
# UWM-CMS: Update Locations Geo Search view to demonstrate proximity integration

This PR changes the Update Locations Geo Search view to demonstrate the ability to filter and sort based on distance from specified location coordinates.

To test:

- Visit /search-geo/locations
- Enter a pair of latitude/longitude coordinates, making sure there are no spaces (example: `47.6237919,-122.3426424`, the coordinates for UW Medicine SLU) and click "Apply"
- Confirm that the results are ordered by the Distance column
- Change the distance value in the form to "5 mi" and click "Apply"
- Confirm that the number of results is reduced from when the value was "20 mi"
- Add a word or phrase (example: `urgent care`) to search the location name and click "Apply"
- Confirm that the results only show locations with "urgent care" in the title
- Confirm that results are still ordered by the Distance column
- Remove the lat/long coordinates and click "Apply"
- Confirm that the results are ordered by the Relevance column and that the Distance column is empty
